pkg/test/e2eutil/wait_util.go

```diff
@@ -97,3 +102,54 @@ func WaitForDeletion(t *testing.T, dynclient client.Client, obj runtime.Object,
 	t.Logf("%s %s was deleted\n", kind, key)
 	return nil
 }
+
+// WaitForCondition checks to see if a given object has a given condition type/status pair, trying every retryInterval until
+// the timeout has elapsed. If the condition is not found before the timeout, the function returns an error.
+func WaitForCondition(t *testing.T, dynclient client.Client, obj runtime.Object, cType status.ConditionType, cStatus v1.ConditionStatus, retryInterval, timeout time.Duration) error {
```

Comment:
Tests have a specific nomenclature were the word Test goes in the name.
I suggest using the IDE feature to gen then because it will gen using the testing lib in the property way and with the correct nomenclature. 


See : Generate ... 
<img width="203" alt="Screenshot 2019-11-19 at 10 13 34" src="https://user-images.githubusercontent.com/7708031/69137761-4c14a080-0ab5-11ea-8047-97361b673e82.png">

Reply:
This isn't a test. It's a utility function that can be used by tests, similar to the other `WaitFor*` functions in this file.

Reply:
Catcher. However, note that this tip s valid for the other places/comments above.